test/apps/rails4/lib/sweet_lib.rb

```diff
@@ -2,4 +2,9 @@ class SweetLib
   def do_some_cool_stuff bad
     `ls #{bad}`
   end
+
+  def test_find_group
+    #Should warn, no escaping done for :group
```

Comment:
Doh...just noticed this after the merge. I made a SQL based test on my firstbpass, but I changed it to be a command exec test and forgot to update the comment. My apologies.